### PR TITLE
Resolve netty version conflicts

### PR DESCRIPTION
**Pull request by jacko9et**

### CODE REVIEW

File: eng/versioning/external_dependencies.txt [hunk 1]
```diff
@@ -38,10 +38,12 @@ io.micrometer:micrometer-core;1.5.4
 io.micrometer:micrometer-registry-azure-monitor;1.5.4
 io.micrometer:micrometer-registry-graphite;1.5.4
 io.netty:netty-buffer;4.1.51.Final
+io.netty:netty-common;4.1.51.Final
 io.netty:netty-codec-http;4.1.51.Final
 io.netty:netty-codec-http2;4.1.51.Final
 io.netty:netty-handler;4.1.51.Final
 io.netty:netty-handler-proxy;4.1.51.Final
+io.netty:netty-transport;4.1.51.Final
```
Comment (chenrujun) [inline eng/versioning/external_dependencies.txt:46]: Please sort lines by alphabet order.

File: sdk/spring/azure-spring-boot-starter-servicebus-jms/pom.xml [hunk 2]
```diff
@@ -40,7 +40,42 @@
...
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-kqueue</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-transport-native-kqueue;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http</artifactId>
+      <version>4.1.51.Final</version> <!-- {x-version-update;io.netty:netty-codec-http;external_dependency} -->
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-servicebus-jms/pom.xml:78]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/scripts/managed_external_dependencies.txt#L868-L898

Do we need to add these all?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-servicebus-jms/pom.xml (hunk 2 above)
Comment (lu-cheng) [reply sdk/spring/azure-spring-boot-starter-servicebus-jms/pom.xml:78]: I only added the netty module that qpid depends on, in order to solve the problem that the netty version that qpid depends on may be inconsistent with the SDK.

**Review by chenrujun — APPROVED** [review]
LGTM. Thank you for your effort.

**Comment by aholler2** [comment]
Does this resolve existing versions of the Azure SDK or will this only affect a new version release?

**Comment by chenrujun** [comment]
@aholler2 

> Does this resolve netty dependency issues with existing versions of the Azure Java SDK, e.g. Storage Blob 12.7.0, or will this only fix new version releases?

This only fix new releases.

**Pull request merged**